.github/workflows/on_push_protected_branches.yml

```diff
@@ -25,8 +25,8 @@ jobs:
         name: "Pull dependencies"
         run: go mod vendor
       - 
-        name: "Unit tests"
-        run: make test
+        name: "Tests"
+        run: make test-integration
```

Comment:
Actually it should be `make test` right?

Reply:
Good catch on integration tests PR :+1: I completely forgot to adjust the protected branches workflow.

Now looking at it, I wonder if it makes sense to copy all these unit tests/integration tests and have the same identical code in 2 workflows. And actually there is an option to run jobs conditionally. Here we could use this to have everything in one workflow and run the release job only for develop and `master branches.

Check latest commit, which implements this approach.